src/features/applications/hooks/slug.ts

```diff
@@ -17,10 +17,14 @@ hooks.addPureHook('POST', 'resin', 'application', {
 				},
 			});
 			if (organization != null) {
-				request.values.slug = getApplicationSlug(
-					organization.handle,
-					request.values.app_name,
-				);
+				try {
+					request.values.slug = getApplicationSlug(
```

Comment:
What do you think about adding a explicit, instead of relying on the try-catch?
```
	if (request.values.app_name == null) {
		throw new BadRequestError/NotFoundError('Missing application name');
	}
```
We can then leave the `throw new Error()`s as is in `getApplicationSlug` as they are, to signal w/ a 500 that we haven't checked the parameters in one of our codepaths.